### DISCUSSION TITLE
Graphic charter

### DISCUSSION THREAD

**Discussion by Yopler** [HandyOrg/HandyControl#1029] [Show and tell]
Hey 🖐, (related to issue #1017)

I have started the graphic charter, but I feel restrain by the actual color (Color.xaml, ect.) and how they are used.

Have the liberty to propose an graphic charter inspired by the actual design where I can "delete" some existing color and add new ones ?

🚨Work In Progress 🚨
https://www.figma.com/file/Pk5MdBHzMPUr0JFLFeE8IS/HandyControl

**Comment by ghost1372** [comment]
Hi, wow thank you
I agree with this, but the decision is with @NaBian

**Reply by Yopler** [reply]
It's currently only the first step, where I just clean the repeat color. 
exemple : 
`PrimaryColor` and `DarkPrimaryColor` always the same.
the `ThirdlyTextColor` don't match the contrast with the `SecondaryRegionColor` (but still ok if it is for hover Trigger) and `ThirdlyRegionColor`.

**Reply by Yopler** [reply]
@ghost1372 , @NaBian 
I have take some liberty (in a new page) about colors (in a new page) to propose the absolute minimum in a color charter.
what I think is missing, at least :
- shadow rules
- smoke backgroung (layer behind a dialog to not access application behind)
- and maybe some other, if you want to have special color for popup (for example) or specific control like card or others

A graphic charter, include also a typography charter (typography use and typography hierarchy)
I can add also a layering rule part, but this is more about design system.

**Reply by Yopler** [reply]
@NaBian @ghost1372 ,🖐

So I have take some liberty ! To make my porposition more immersive ! I have create 2 figma components : Button and ComboBox (The behavior of the ComboBox : item we can select more than one , that is not the actual default behavior of combobox, but I can not do better using figma ! )
You can find the prototype here : [test page](https://www.figma.com/proto/Pk5MdBHzMPUr0JFLFeE8IS/HandyControl?node-id=94%3A1521&scaling=contain&page-id=75%3A764) 

I will not go further until Nabian give me his return / decision ! 😄

Hope that will help you, Nabian ! 😉
